sirepo/package_data/template/srw/rsoptExport.sh.jinja

```diff
@@ -1,11 +1,27 @@
 #!/bin/bash
-echo "NOTE: remove files from {{ runDir }} first"
+if [ -d "{{ runDir }}" ] && [ -n "$(ls -A {{ runDir }})" ]; then
+   echo "Cleaning run directory {{ runDir }}..."
+   rm {{ runDir }}/{{ pyFileName }}
+   rm {{ runDir }}/run_parallel_python.py
+fi
+echo "Generating data..."
 SECONDS=0
 rsopt sample configuration "{{ ymlFileName }}" > "{{ outFileName }}" 2>&1
 if [ $? -eq 0 ]
 then
-    echo "Completed scan in $SECONDS sec"
-    echo "NOTE: python {{ pyFileName }} rsopt_run" `tail -1 "{{ outFileName }}" | cut -d: -f2`.npy "to run SRW with generated data"
+    echo "Completed generation in $SECONDS sec"
```

Comment:
This is cool. I didn't know about `$SECONDS`. However, I wouldn't bother printing it. It's an unnecessary output. There's a prompt below which is all that's needed.

Reply:
5421ff1